addons/im_livechat/static/src/core/common/res_users_model_patch.js

```diff
@@ -0,0 +1,9 @@
+import { patch } from "@web/core/utils/patch";
+import { ResUsers } from "@mail/core/common/res_users_model";
+
+patch(ResUsers.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.is_livechat_manager = false;
+    },
+});
```

Comment:
I prefer the patch syntax where we define the patch object first so we can give it the correct this type doc.